test/utils/file_utils.py

```diff
@@ -16,9 +16,12 @@
 import tarfile
 
 
-def make_tarfile(script, model, output_path, filename="model.tar.gz"):
+def make_tarfile(script, model, output_path, filename="model.tar.gz", script_path=None):
```

Comment:
default to empty string and then later you can simplify code (plus it matches the type expected):

```suggestion
def make_tarfile(script, model, output_path, filename="model.tar.gz", script_path=""):
```